src/SecurityChecker/Command/SecurityCheckerCommand.php

```diff
@@ -85,13 +125,17 @@ protected function changeFileMode(): void
     }
 
     /**
+     * @param string[] $parameters
+     *
      * @throws \Symfony\Component\Process\Exception\ProcessFailedException
      *
      * @return string
      */
-    protected function runCommand(): string
+    protected function runCommand(array $parameters): string
     {
-        $process = new Process([static::FILE_NAME, 'check:security']);
+        $commands = array_merge([static::FILE_NAME], $parameters);
```

Comment:
It's not a list a commands, but a single command. Please rename variable accordingly